crates/db-address-space/Cargo.toml

```diff
@@ -14,5 +14,5 @@ arc-swap = ">=0.4.8"
 libc = "0.2.39"
 nix = "0.15"
 thiserror = "1"
-vmm-sys-util = "0.8"
+vmm-sys-util = "0.9.0"
```

Comment:
“0.9.0“ limits vmm-sys-util to 0.9.x, is that expected?

Reply:
Yes.
According to the previous talk with @ZizhengBian , we'll describe the dependencies in dragonball-sandbox as a.b.c.

We think a.b.c could help us fix the dependency version in the version we want and a possible future bugfix version for a.b.c
If there is any feature we need for a newer version of that dependency, we could change all a.b.c version number together.